README.md

```diff
@@ -11,27 +11,30 @@ A tool for cross compiling ROS2 packages.
 
 #### Ubuntu
 
-The cross compilation toolchain and docker have to be installed.
+Qemu and Docker have to be installed for cross_compile to work.
 The following instructions have been tested on Ubuntu Xenial (16.04) and Bionic (18.04).
 
 ```bash
-# Install cross compilation toolchain
-sudo apt-get update
-sudo apt-get install -y build-essential cmake git wget curl lsb-core bash-completion \
-    qemu-user-static python3-pip htop g++-arm-linux-gnueabihf 
-# If you are on a non-arm system
-sudo apt-get install -y g++-aarch64-linux-gnu 
-sudo python3 -m pip install -U  colcon-common-extensions rosdep vcstool
-
-# Also install docker and make it available to the current user
-sudo apt-get install -y apt-transport-https ca-certificates curl gnupg-agent software-properties-common
+# Install requirements for Docker and qemu-user-static
+sudo apt update && sudo apt install -y \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    gnupg-agent \
+    lsb-release \
+    software-properties-common \
+    qemu-user-static
+
 curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
 sudo apt-key fingerprint 0EBFCD88
 sudo add-apt-repository "deb [arch=amd64,arm64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
-sudo apt-get update
-sudo apt-get install -y docker-ce
+sudo apt update && sudo apt install -y docker-ce
 sudo usermod -aG docker $USER
-newgrp docker # this reloads the group permissions in the current shell, unnecessary after relogin
+
+# Reload the group permissions in the current shell. Otherwise logout and login again to apply permissions
+newgrp docker
+
+# Verify current user can run Docker
 docker run hello-world
```

Comment:
I think we should avoid reproducing the Docker doc here. Could we just point to https://docs.docker.com/install/linux/docker-ce/ubuntu/#install-docker-engine---community ?

I like `newgrp` though, didn't know about it :)

Reply:
This is a lot more condensed than the Docker docs and includes steps on allowing the current user run Docker containers.

What if I link the Docker docs in addition?

Reply:
sgtm, let me look at the content more carefully in this case.